tests/odoo9/Dockerfile

```diff
@@ -2,3 +2,14 @@ FROM tecnativa/odoo-base:9.0
 
 # Keep repositories status if testing several times
 VOLUME /opt/odoo/custom/src
+
+# Metadata
+ARG VCS_REF
+ARG BUILD_DATE
+ARG VERSION
+LABEL org.label-schema.schema-version="$VERSION" \
+      org.label-schema.vendor=Tecnativa \
+      org.label-schema.license=Apache-2.0 \
+      org.label-schema.build-date="$BUILD_DATE" \
+      org.label-schema.vcs-ref="$VCS_REF" \
+      org.label-schema.vcs-url="https://github.com/Tecnativa/docker-odoo-base"
```

Comment:
It's part of the build scripts, and is required. I never considered a use case where a child Dockerfile would be used, so the fact that the script fills these in never became an issue until now. It's another build tools change that I don't quite have time for TBH - this doesn't necessarily hurt anything & we most certainly use this aspect in our other builds.